src/kirby/components/floating-action-button/floating-action-button.component.scss

```diff
@@ -3,10 +3,25 @@
 // Ionic fab can be customized by using the css properties listed in the CSS Custom Properties section of the below docs page:
 // https://ionicframework.com/docs/api/fab-button
 
+@mixin fab-size($fab-size) {
+    width: #{$fab-size}px;
```

Comment:
Pretty sure the `size()` function returns px-values...

Reply:
unfortunately it no longer works if i don't specify it's a px-value